pydeck: Update installation docs (#5270)
visgl/deck.gl: bindings/pydeck/docs/installation.rst

Getting a Mapbox API key
^^^^^^^^^^^^^^^^^^^^^^^^

> By default, pydeck 0.6 provide basemap tiles through Carto.

Reviewer: ```suggestion
By default, pydeck 0.6 provides basemap tiles through Carto.
```

> You can use a Mapbox API key, which you can do by 

Reviewer: ```suggestion
You can optionally use a Mapbox API key, by 
```
